src/Renderer/ThreeExtended/Feature2Mesh.js

```diff
@@ -96,24 +96,27 @@ function addExtrudedPolygonSideFaces(indices, length, ring, isClockWise) {
     // add indices to make two triangle, that make the side face
     const mod = ring.offset + ring.count + length;
     for (let i = ring.offset; i < ring.offset + ring.count; ++i) {
+        const i1 = Math.max((i + 1) % mod, ring.offset);
+        const i2 = Math.max((i + length) % mod, ring.offset);
+        const i3 = Math.max((i + length + 1) % mod, ring.offset);
```

Comment:
Suggestion: `i` and `i1` could be `floor1`/`floor2` and `i2/i3` could be `ceil1` `ceil2` (or `roof1`..) ?

Reply:
Thanks ! Even maybe `bottom1/2` and `top1/2` as floor/ceil/roof is vocabulary for buildings.